src/System.Collections/tests/Generic/SortedSet/SortedSet.TreeSubSet.Tests.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace System.Collections.Tests
+{
+    public class SortedSet_TreeSubset_Int_Tests : SortedSet_TreeSubset_Tests<int>
+    {
+        protected override int Min => int.MinValue;
+        protected override int Max => int.MaxValue;
+        private int Current { get; set; } = 1;
```

Comment:
It's a private property, not a field.  By adding the { get; set; }, you've made it a property with the compiler providing an implicit backing property and implicitly implemented getters and setters.  That's all unnecessary, when it could instead just be ```private int _current = 1;```